src/psyclone/tests/parse/module_info_test.py

```diff
@@ -431,48 +431,50 @@ def test_module_info_coverage_source_node_found(tmpdir, monkeypatch):
     assert "Could not find file" in str(einfo.value)
 
 
-def test_module_info_coverage_fparser_tree(tmpdir, monkeypatch):
+def test_module_info_coverage_fparser_error(tmpdir, monkeypatch):
     """
-    Coverage test: Fparser
+    Coverage test:
+    - Create an .f90 file with wrong syntax
+    - Test for raised Exception if creating fparser tree.
     """
 
     filename = os.path.join(tmpdir, "testfile_module_info_a.f90")
 
-    #
-    # Get fparser
-    #
     with open(filename, "w", encoding='utf-8') as fout:
         fout.write(SOURCE_DUMMY)
 
-    node = Node("Dummy")
     module_info: ModuleInfo = ModuleInfo(
-            "my_mod", FileInfo(filename), node)
+            "my_mod", FileInfo(filename))
 
     module_info.get_fparser_tree()
 
     #
-    # Error in parsing source code with fparser
+    # Create error in source code
     #
     with open(filename, "w", encoding='utf-8') as fout:
         fout.write(SOURCE_DUMMY+"\ncreate some error")
 
-    node = Node("Dummy")
     module_info: ModuleInfo = ModuleInfo(
-            "my_mod", FileInfo(filename), node)
+            "my_mod", FileInfo(filename))
 
     with pytest.raises(ModuleInfoError) as einfo:
         module_info.get_fparser_tree()
 
-    assert "ModuleInfoError: Error parsing" in str(einfo.value)
+    assert ("ModuleInfoError: Error(s) getting fparser tree of file"
+            in str(einfo.value))
```

Comment:
This is good. In line with one of my previous comments, please could you check that calling module_info.get_fparser_tree() a second time doesn't result in another attempt to parse the file. (Although it should still raise the same error or perhaps it would be better if it returned None rather than raising an error?)

Reply:
I added it.